census/estimator/trainer/model.py

```diff
@@ -202,7 +201,7 @@ def csv_serving_input_fn():
   )
   features = parse_csv(csv_row)
   features.pop(LABEL_COLUMN)
-  return tf.contrib.learn.InputFnOps(features, None, {'csv_row': csv_row})
+  return tf.estimator.export.ServingInputReceiver(features, {'csv_row': csv_row})
```

Comment:
Does this make sense/is necessary with the new APIs?

Reply:
That's a receiver Tensor (second arg). Sorry not sure I understand the comment?